doc/examples/segmentation/plot_rolling_ball.py

```diff
@@ -274,8 +270,8 @@ def plot_result(image, background):
 # Smoother peaks can be removed with smaller values of the radius.
 
 x = pywt.data.ecg()
-background = restoration.rolling_ball(x, radius=80)
-background2 = restoration.rolling_ball(x, radius=10)
+background = ski.restoration.rolling_ball(x, radius=80)
+background2 = ski.restoration.rolling_ball(x, radius=10)
 plt.figure()
```

Comment:
Should update to Matplotlib's OO style here.

Reply:
If you feel like it, but orthogonal to the issue I think. :)

Reply:
Yes, totally orthogonal to this PR! I meant, it should be addressed in a different PR. In #7531, @GParolini took care of this for a few examples in `doc/examples/segmentation` but not all of them.